app/src/main/java/com/example/superheroapi/model/SuperHeroes.kt

```diff
@@ -30,7 +30,6 @@ data class PowerStats(
     val intelligence: String,
     val strength: String,
     val speed: String,
-    val aliases: List<String>,
```

Comment:
aliases was giving a serialization error, so I believe it actually wasn't part of the PowerStats json.